build/config.gni

```diff
@@ -45,10 +45,14 @@ declare_args() {
     # Generate all symbols (breakpad + native) for Release builds
     brave_debug_symbol_level = 2
   } else {
-    brave_debug_symbol_level = 1
+    brave_debug_symbol_level = 0
   }
 }
 
+declare_args() {
+  should_generate_breakpad_symbols = brave_debug_symbol_level != 0
```

Comment:
nit: I don't think this needs to be declare args, does it? It's just for internal like brave_version below